definitions.yaml

```diff
@@ -237,6 +237,114 @@ definitions:
         type: object
         additionalProperties: true
 
+  V4GetClusterStatusResponse:
+    type: object
+    description: Object about a cluster's current state
+    properties:
+      aws:
+        type: object
+        description: |
+          AWS specific status information. Only given in AWS installations.
+        properties:
+          availabilityZones:
+            type: array
+            items:
+              type: object
+              properties:
+                name:
+                  type: string
+                  description: Name of the randomly selected availability zone
+                subnet:
+                  type: object
+                  properties:
+                    private:
+                      type: object
+                      properties:
+                        cidr:
+                          type: string
+                          description: |
+                            Private subnet CIDR allocated for this availability
+                            zone
+                    public:
+                      type: object
+                      properties:
+                        cidr:
+                          type: string
+                          description: |
+                            Public subnet CIDR allocated for this availability
+                            zone
+      cluster:
+        type: object
+        properties:
+          conditions:
+            type: array
+            items:
+              type: object
+              description: TODO
```

Comment:
How about: "List of cluster conditions. Cluster conditions describe the state transitions of a cluster (eg. cluster creation, upgrade and deletion)."

Maybe also giving an example of a condition here. Eg a cluster was created or is currently upgraded.

Reply:
I added some brief description, but as above the example is missing. Please suggest something.